vcf_view.wdl

```diff
@@ -48,6 +48,10 @@ task xVCFView {
         preemptible: "${preemptible}"
     }
     command <<<
+        # do not delete these lines -- they are required for indexing to work properly on Terra
+        set -eux -o pipefail
+        find . -type d -exec sudo chmod -R 777 {} +
```

Comment:
This seems a bit like overkill.  Do you know which file needs to be modified with what permissions?

Reply:
It's not a file that needs modification, but rather the directory. You can confirm this locally -- bcftools index works fine on a read-only vcf.gz file. 

The `-type d` in `find` will only return directories, which on Terra will include the directory containing input files. Indexing a VCF file doesn't modify the VCF, but it drops an index file into the same directory as the VCF file.

There is another way of doing this, but it's more complicated. I'll open a fresh PR if it passes on Terra.